Sources/Logger/Loggers/FileLogger/FileLogger.swift

```diff
@@ -52,36 +59,56 @@ public class FileLogger: Logging {
     }
 
     var currentLogFileUrl: URL {
-        suiteName == nil ?
-            logDirURL.appendingPathComponent("\(currentLogFileNumber)").appendingPathExtension(logFilePathExtension) :
-            logDirURL.appendingPathComponent("extension-\(currentLogFileNumber)").appendingPathExtension(logFilePathExtension)
+        if let appName = appName {
+            return logDirURL
+                .appendingPathComponent("\(appName)-\(currentLogFileNumber)")
+                .appendingPathExtension(logFilePathExtension)
+        } else {
```

Comment:
Kdyz ma `if` v sobe `return` tak nemusis davat `else`. Je to akorat dalsi zbytecne vnorenej kontext.

Reply:
Opraveno.